src/main/webapp/js/instructor.es6

```diff
@@ -369,6 +369,25 @@ function attachEventToDeleteStudentLink() {
     });
 }
 
+function remindButtonAjax(urlLink) {
```

Comment:
The name sounds accurate enough, as it sends a POST request to the linked URL (which in the older version would be retrieved via GET ie. the browser is redirected to that link). The resulting page is then retrieved, and the status message is extracted out from the resulting page, and displayed on the current one.

While this solution might sound a little "hackish", since an entire page is returned just to extract out one status message, it possesses an advantage: if a user was to access the URL manually (for instance by right-clicking "Remind all students", followed by "Open link in new tab/window", as ctrl-clicking would open the modal as normal), a complete page would be returned instead of just a single status message.

Of note however is that emails will be sent out without confirmation (no modal) if a user opts to open the link via this method (right-clicking followed by "new tab/window"), provided the referrer and session token are correct. This behaviour exists on the `master` branch as well, and this may need to be fixed in a PR if it is found to be problematic.